example.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
```

Comment:
It would be nice to turn this into a test case so that we don't reintroduce the issue accidentally.